web/jobs/jobs.go

```diff
@@ -108,10 +131,72 @@ func pushJob(c echo.Context) error {
 	return nil
 }
 
+func newTrigger(c echo.Context) error {
+	instance := middlewares.GetInstance(c)
+	scheduler := instance.JobsScheduler()
+	req := &apiTriggerRequest{}
```

Comment:
> Why do you use an intermediate apiTriggerRequest and can't deserialize directly to a jobs.TriggerInfos?

They do not share the exact same properties regarding the worker arguments. This allow to have a `bytes.RawMessage` and not having to parse the worker arguments on trigger creation and delay it at execution.

> You don't use JSON-API for the requests, only for the responses?

Indeed I need to fix that. The lazy part of myself hid it from me.